app/javascript/pages/country/widget/widgets/widget-fao-extent/widget-fao-extent.js

```diff
@@ -0,0 +1,107 @@
+import { createElement, PureComponent } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import isEqual from 'lodash/isEqual';
+import { format } from 'd3-format';
+import {
+  getPeriods,
+  getActiveFilter
+} from 'pages/country/widget/widget-selectors';
+
+import WidgetTreeCoverGainComponent from './widget-fao-extent-component';
+import actions from './widget-fao-extent-actions';
+
+export { initialState } from './widget-fao-extent-reducers';
+export { default as reducers } from './widget-fao-extent-reducers';
+export { default as actions } from './widget-fao-extent-actions';
+
+const PERIODS_LIST = [
```

Comment:
Can this be put in data folder along with all the other constants for selectors?